addons/web/static/src/views/list/list_renderer.js

```diff
@@ -832,7 +832,10 @@ export class ListRenderer extends Component {
 
     getFormattedValue(column, record) {
         const fieldName = column.name;
-        if (column.options.enable_formatting === false) {
+        if (
+            column.options.enable_formatting === false &&
+            !["integer", "float"].includes(column.fieldType)
+        ) {
```

Comment:
I am not sure for that. I think it would be better to add additional if for the special case we got with `id` field. 🤔

Reply:
Actually, do we really have the issue in that case? I think we will only have the issue when we create a record and we display the id column in the list view, no?